components/Navbar.vue

```diff
@@ -1,41 +1,60 @@
 <template>
-  <header class="navbar">
-    <nav>
-      <div class="box">
-        <div class="logo">
-          <a href="https://realdevsquad.com/">
+  <header class="wrapper">
+    <nav class="navBar">
+      <div class="hamburger" @click="toggleClicked(isClicked)">
+        <img src="~assets/menu.png" />
+      </div>
+      <div class="navBarLogin">
+        <a
+          class="btnLogin"
+          href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+        >
+          <button type="button" class="btnLoginText">
+            Sign In With GitHub
             <img
-              src="~/assets/Real-Dev-Squad@1x.png"
-              alt="Real Dev Squad logo"
-            />
-          </a>
-        </div>
-        <div class="hamburger-menu" @click="toggleClicked(isClicked)">
-          <span></span>
-          <span></span>
-          <span></span>
-        </div>
+              className="githubLogo"
+              src="~/assets/github.png"
+              alt="GitHub Icon"
+              height="15px"
+              width="15px"
+            /></button
+        ></a>
+      </div>
+
+      <div class="navbarMenuItem">
+        <ul :class="[isClicked ? 'hide-menu' : 'navBarMenu']">
+          <li class="navBarLogoLi">
+            <a href="https://realdevsquad.com/">
+              <img
+                src="~/assets/Real-Dev-Squad@1x.png"
+                alt="Real Dev Squad logo"
+                width="50px"
+                height="50px"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="https://welcome.realdevsquad.com/">Welcome</a>
+          </li>
+          <li>
+            <a href="https://www.realdevsquad.com/events.html">Events</a>
+          </li>
+          <li>
+            <a href="https://members.realdevsquad.com/">Members</a>
+          </li>
+          <li>
+            <a href="https://crypto.realdevsquad.com/">Crypto</a>
+          </li>
+          <li>
+            <a href="https://status.realdevsquad.com/">Status</a>
+          </li>
+          <li>
+            <a class="activeTab" href="https://goals.realdevsquad.com/"
+              >Goals</a
+            >
+          </li>
```

Comment:
Can you please avoid the repetition of code by looping through an object of navbar links and their name